hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java

```diff
@@ -101,6 +101,11 @@ private static HugeGraph initGraph(String configPath) throws Exception {
                 LOG.info("Skip init-store due to the backend store of '{}' " +
                          "had been initialized", graph.name());
```

Comment:
@Z-HUANT seems we could move this log after L108? (due to the code logic)

could enhance it in another reviewed PR

Reply:
I'll put it in this PR later: https://github.com/apache/incubator-hugegraph/pull/2408.